contracts/system_addresses.go

```diff
@@ -0,0 +1,25 @@
+package contracts
+
+import "github.com/0xPolygon/polygon-edge/types"
+
+var (
+	// ValidatorSetContract is an address of validator set contract deployed to child chain
+	ValidatorSetContract = types.StringToAddress("0x101")
+	// BLSContract is an address of BLS contract on the child chain
+	BLSContract = types.StringToAddress("0x102")
+	// MerkleContract is an address of Merkle contract on the child chain
+	MerkleContract = types.StringToAddress("0x103")
+	// StateReceiverContract is an address of bridge contract on the child chain
+	StateReceiverContract = types.StringToAddress("0x1001")
+	// NativeTokenContract is an address of bridge contract (used for transferring native tokens on child chain)
+	NativeTokenContract = types.StringToAddress("0x1010")
+	// SystemCaller is address of account, used for system calls to smart contracts
+	SystemCaller = types.StringToAddress("0xffffFFFfFFffffffffffffffFfFFFfffFFFfFFfE")
+
+	// NativeTransferPrecompile is an address of native transfer precompile
+	NativeTransferPrecompile = types.StringToAddress("0x2020")
+	// BLSAggSigsVerificationPrecompile is an address of BLS aggregated signatures verificatin precompile
+	BLSAggSigsVerificationPrecompile = types.StringToAddress("0x2030")
+	// ConsolePrecompile is and address of Hardhat console precompile
+	ConsolePrecompile = types.StringToAddress("0x000000000000000000636F6e736F6c652e6c6f67")
```

Comment:
Just curious. Is it required?

Reply:
Not sure what are you referring to? Is it about this `ConsolePrecompile` we've added, or is it in general these addresses which we have put there as global variables?

Anyways I'll try to guess that you were referring to `ConsolePrecompile`:
it must be strictly deployed to the given address, because it intercepts Hardhat console logs (which turned out to be pretty handy when troubleshooting smart contracts execution). :slightly_smiling_face:

Reply:
Yes, I meant contract. I was not sure why you needed it. Thanks

Reply:
It has proven multiple times that it is good to have console precompile when troubleshooting SC execution :smile: However, since it is a debugging tool, we can figure out in the future whether it makes sense to distribute it to production system (either remove it completely before release or deliver it with some custom build tags)...